apps/charterafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,176 @@
+import link from "../fields/link";
+import richText from "../fields/richText";
+import {
+  validateHexColor,
+  blendModeOptions,
+  MuiButtonColors,
+} from "../utils/colors";
+
+const Hero = {
+  slug: "hero",
+  fields: [
+    {
+      name: "slides",
+      type: "array",
+      minRows: 1,
+      fields: [
+        {
+          name: "title",
+          type: "group",
+          required: true,
+          fields: [
+            {
+              name: "color",
+              type: "text",
+              validate: validateHexColor,
+              required: true,
+              label: {
+                en: "Title Color",
+                fr: "Couleur du titre",
+                pt: "Cor do título",
+              },
+            },
+            richText({
+              name: "content",
+              required: true,
+              admin: {
+                elements: ["h2", "h3", "h4", "h5", "h6", "link"],
+                leaves: ["bold", "italic", "underline", "code"],
+              },
+              localized: true,
+              label: {
+                en: "Title",
+                fr: "Titre",
+                pt: "Título",
+              },
+            }),
```

Comment:
```suggestion
              name: "content",
              label: {
                en: "Title",
                fr: "Titre",
                pt: "Título",
              },
              required: true,
              localized: true,
              admin: {
                elements: ["h1"],
                leaves: ["bold", "italic", "underline", "code"],
              },
            }),
```

1. `name`, `label`, `fields`, `required`, `localized`, others (including `admin`) is a good order of field props. Otherwise, it's hard to see if `label` fits the `name`, etc.
2. I don't think we need `admin.elements` here. Everything needs to go into h1 on the front end. Not sure how you'd handle inserting `<br />`.